src/main/scala/nl/knaw/dans/easy/stage/fileitem/EasyStageFileItem.scala

```diff
@@ -59,7 +58,10 @@ object EasyStageFileItem {
           warning.map(msg => log.warn(msg))
           val rows = csv.getRows
           if (rows.isEmpty) log.warn(s"Empty CSV file")
-          rows.map(options => FileItemSettings(options ++ trailArgs))
+          rows.map{options =>
+            log.info(options.mkString(" "))
```

Comment:
Add some explanatory prefix before the options. Just dumping the options to the log without any explanation may come across cryptic.